opentracing-util/src/main/java/io/opentracing/util/AutoFinishScope.java

```diff
@@ -19,6 +19,17 @@
 
 import java.util.concurrent.atomic.AtomicInteger;
 
+/**
+ * @deprecated use {@link ThreadLocalScope} instead.
+ * The operation mode of this class contrasts with the 0.32
+ * deprecation of auto finishing {@link Span}s upon {@link Scope#close()}.
```

Comment:
Could also add a link to this issue for more background.

Reply:
Great idea!

Reply:
Done ;)